src/applications/caregivers/components/NeedHelpFooter/index.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+const NeedHelpFooter = () => (
+  <footer className="need-help-footer row">
+    <h5>Need Help?</h5>
+    <hr />
+    <span>Need help filling out this form? Call our toll-free number: </span>
+
+    <section>
+      <a href="tel:18772228387">1-877-222-8387</a>
+      <span>
+        TTY: <a href="tel:18008778339">1-800-877-8339</a>
+      </span>
+      <span>Monday — Friday, 8:00am — 8:00pm (ET)</span>
+    </section>
+
+    <section>
+      <span>A Caregiver Support Coordinator locator is available at: </span>
+      <a href="http://www.caregiver.va.gov/">http://www.caregiver.va.gov/</a>
```

Comment:
Do we want `rel="noopener noreferrer"` on this a tag? Also we may want to change it to `https` if that's possible!

Reply:
I thought it was a good practice to always add them for security, I  can defiantly add HTTPS just copied and pasted from design mocks-ups.